lib/components/popups/course-search/Cart.tsx

```diff
@@ -169,64 +169,17 @@ const Cart: FC<{ allCourses: SISRetrievedCourse[] }> = (props) => {
     extras: SearchExtras,
   ): Promise<[SISRetrievedCourse[], number[]]> => {
     return new Promise(async (resolve) => {
-      // let courses: SISRetrievedCourse[] = [...courseCache]; // how actively is this cache updated?
-      // if (!retrievedAll) { // how often is this filter used?
       const retrieved: any = await axios
-        .get(getAPI(window) + '/search', {
+        .get(getAPI(window) + '/cartSearch', {
           params: getParams(extras),
```

Comment:
do same old search, just renamed to `/cartSearch` (because new search route isn't ready to handle prereq or fine requirements course search)